backend/api/Models/Enums/InspectionType.cs

```diff
@@ -0,0 +1,21 @@
+﻿public class InspectionType
+{
+    private InspectionType(string value) { Value = value; }
+
+    public string Value { get; private set; }
+
+    public static InspectionType Image { get { return new InspectionType("image"); } }
+    public static InspectionType ThermalImage { get { return new InspectionType("thermal_image"); } }
+    public static InspectionType Audio { get { return new InspectionType("audio"); } }
+
+    public static InspectionType EchoSensorTypeToInspectionType(string sensorType)
+    {
+        return sensorType switch
+        {
+            "Picture" => Image,
+            "ThermicPicture" => ThermalImage,
+            "Audio" => Audio,
+            _ => Image,
+        };
+    }
+}
```

Comment:
Convert to Enum, string conversion is taken care automatically by:
`builder.Services.AddControllers().AddJsonOptions(options => options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter()));`

Reply:
Fixed